tests/src/index.json

```diff
@@ -400,7 +400,19 @@
     "name" : "org.example:library",
     "versions" : [ "0.0.1" ]
   } ]
-}, {
+},
+  {
```

Comment:
This metadata file doesn't look correctly formatted, but passed all the CI checks. Is there something broken?

Reply:
It looks like only this new metadata is not formatted properly. Maybe CI checks pass because this is also correctly formatted but with different style. Did you do something different compared to your previous contributions?

Reply:
No. I tried to run the JSON reformat, but nothing changed.

Reply:
Could [this](https://github.com/oracle/graalvm-reachability-metadata/pull/532) be the reason why CI passes now?

Reply:
I have restored the code before the change and wrongly formatted files still pass. There's something else broken.